code/modules/antagonists/morph/morph.dm

```diff
@@ -181,10 +186,13 @@
 
 /mob/living/simple_animal/hostile/morph/can_track(mob/living/user)
 	if(morphed)
-		return 0
+		return FALSE
 	return ..()
 
 /mob/living/simple_animal/hostile/morph/AttackingTarget()
+	if(!melee_damage_disguised)
```

Comment:
if it tries to attack with 0 damage it fails and doesn't allow it to nuzzle people. can change this if people want although i feel like this is a newbie trap if it let people attack only to find out it doesn't do anything and promptly get lynched.